src/guide/effects.mdx

```diff
@@ -0,0 +1,498 @@
+---
+title: "Graphical Effects"
+index: 7
+---
+
+@@QtQuick.Effects.MultiEffect lets you apply multiple visual effects—like blur, shadow,
+and color adjustments—in a single, easy-to-use QML component.
+
+# Qt 6 MultiEffect
+
+Now, how do I use this you ask? Well, this guide provides a few examples for common uses of
+effects such as drop shadows, colourisation, masks and blurring.
+
+First and foremost (and arguably the most commonly used), drop shadows! To create a drop shadow,
+the [QtQuick.Effects](https://doc.qt.io/qt-6/qtquick-effects-qmlmodule.html) module provides
+two components that you can use: @@QtQuick.Effects.RectangularShadow and @@QtQuick.Effects.MultiEffect.
+
+@@QtQuick.Effects.RectangularShadow is much easier to use and more efficient than
+@@QtQuick.Effects.MultiEffect, however it can only be used for rectangular shadows (as the name implies).
+On the other hand, @@QtQuick.Effects.MultiEffect can used for most common effects, including all of
+the previously mentioned effects.
```

Comment:
Cut the bit about shadows. We'll leave that to the headers which you can go to to learn about each thing.